packages/eslint-config/.eslintrc.js

```diff
@@ -0,0 +1,66 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/eslint-config
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * Default ESLint configuration for LoopBack
+ *
+ * See https://eslint.org/docs/user-guide/configuring
+ */
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  plugins: ['eslint-plugin', '@typescript-eslint'],
+  env: {
+    es6: true,
+    node: true,
+  },
+  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
+  rules: {
+    'comma-dangle': ['error', 'always-multiline'],
+    'no-mixed-operators': 'error',
+    'no-console': 'off',
+    'no-undef': 'off',
```

Comment:
What is this rule about and why are we turning it off, against the recommendations?